tests/runtime/call

```diff
@@ -425,7 +425,6 @@ PROG BEGIN { print("BEGIN"); @["a"] = avg(10); @["b"] = avg(20); @["c"] = avg(30
 EXPECT BEGIN
        @[c]: 3
        @[d]: 4
-       
```

Comment:
Because 'avg' is now on the same path as other maps, this extra space was removed. I can add it back but wasn't sure if I should.